cli.js

```diff
@@ -0,0 +1,54 @@
+const { validate, validateStats} = require('./validate');
+const { getLinks, uniqueLinks,stats} = require('./stats');
+const fs = require("fs")
+const path = require('path');
+const marked = require('marked');
```

Comment:
Por nomenclatura es recomendable tener las dependencias en la parte superior
```suggestion
const fs = require("fs")
const path = require('path');
const marked = require('marked');
const { validate, validateStats} = require('./validate');
const { getLinks, uniqueLinks,stats} = require('./stats');
```